### PR TITLE
Post policy hidden eye in speedgrader

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-11703
affects: Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: rn/Teacher/src/modules/speedgrader/__tests__/SpeedGrader.test.js [hunk 1]
```diff
@@ -83,11 +84,13 @@ let defaultProps = {
   assignmentSubmissionTypes: ['none'],
   gradeSubmissionWithRubric: jest.fn(),
   getCourseEnabledFeatures: jest.fn(),
+  getEnabledFeatureFlags: jest.fn(() => Promise.resolve({ data: [] })),
 }
 
 describe('SpeedGrader', () => {
   beforeEach(() => {
     jest.clearAllMocks()
+    SpeedGrader.drawerState = new DrawerState()
```
Comment (shichongrui) [inline rn/Teacher/src/modules/speedgrader/__tests__/SpeedGrader.test.js]: Reseting the `drawerState` on each test is what caused the giant change in the snapshot.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1004971&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Teacher

[MBL-11703](https://instructure.atlassian.net/browse/MBL-11703)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 95.06% | 95.06% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**